### PR TITLE
Pins created on click with label. Route created on button click

### PR DESCRIPTION
**Pull request by rskidmore1**
deepin-screen-recorder_chromium-browser_20210909172722

### CODE REVIEW

File: client/pages/routing.jsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+  }
+
+  makeRouteURL() {
+    let link = 'https://www.google.com/maps/dir/';
+    for (let i = 0; i < this.state.route.length; i++) {
+      link = `${link}${this.state.route[i].coords.lat},${this.state.route[i].coords.lng}/`;
+    }
+    // eslint-disable-next-line no-console
+    console.log(link);
```
Comment (thebearingedge) [inline client/pages/routing.jsx]: If you don't need this anymore, please delete it.

File: client/pages/routing.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      .then(result => {
+        this.setState({ vehicle: result });
+      });
+
+  }
+
+  onMapClicked(mapProps, map, clickEvent) {
+
+    const eventRaw = JSON.stringify(clickEvent);
+    const event = JSON.parse(eventRaw);
```
Comment (thebearingedge) [inline client/pages/routing.jsx]: Is there a specific reason to stringify and then immediately parse the same data?

**Review by thebearingedge — CHANGES_REQUESTED** [review]
Were you planning on storing your stops in your database? If not, that's fine. But if so, they should be stored during this feature if possible.

Please see my other notes.

File: client/pages/routing.jsx (hunk 1 above)
Comment (rskidmore1) [reply client/pages/routing.jsx]: I needed this to show the link created

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**